.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ references:
         GITHUB_TEAM_NAME_SLUG: laa-apply-for-legal-aid
     - image: postgres:10.5
     - image: redis:5.0
-    - image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-apply-for-legal-aid/clamav:1.0.0
+    - image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/laa-apply-for-legal-aid/clamav:1.0.2
```

Comment:
`clamav`'s repo has also been migrated to `live-1` and we are now pulling the image from the ECR repo on `live-1`
I tried pulling a `live-0` image from `live-1` but it didn't work.